src/components/Table/Table.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC } from 'react';
 import { useTable, usePagination, useExpanded } from 'react-table';
 import { css } from 'emotion';
 import { useStyles } from '@grafana/ui';
-import { Overlay } from 'components';
+import { Overlay } from '..';
```

Comment:
```suggestion
import { Overlay } from '../components';
```

Not sure if it's the right path but something like this would be more explicit

Reply:
The development environment tells me that this will be such a solution, so I fixed it a little differently
![image](https://user-images.githubusercontent.com/90199600/141778918-ae84c34c-f3a5-4dc8-9f63-cd85b7b1a8ad.png)

Reply:
Awesome just put it in singlequotes to fix the linting

Reply:
Fixed